build.gradle.kts

```diff
@@ -59,6 +61,9 @@ dependencies {
   testImplementation("org.springframework.boot:spring-boot-starter-test")
   testImplementation("org.junit.jupiter:junit-jupiter-api")
   testImplementation("org.junit.jupiter:junit-jupiter-params")
+  testImplementation("com.ekino.oss.jcv:jcv-core:$jcvVersion")
+  testImplementation("com.ekino.oss.jcv:jcv-hamcrest:$jcvVersion")
```

Comment:
You can just add `jcv-hamcrest` while the core module is a transitive dependency.